howto/pythia.md

```diff
@@ -0,0 +1,20 @@
+## Pythia weights
+
+The process to use Pythia weights is the same as described in our [other guides](download_weights.md), just with [modified arguments](customize_paths.md)
+
+For instance, to run inference with the [pythia-70m](https://huggingface.co/EleutherAI/pythia-70m) checkpoint:
+
+```bash
+python scripts/download.py --repo_id EleutherAI/pythia-70m --local_dir checkpoints/hf-pythia/pythia-70m
+
+python scripts/convert_hf_checkpoint.py --output_dir checkpoints/lit-pythia/pythia-70m --ckpt_dir checkpoints/hf-pythia/pythia-70m
+
+python generate.py --prompt "Hello, my name is" \
```

Comment:
This is very verbose. Since we are generating these files ourselves during conversion, can't we just point generate to a file or directory and discover the other files by default?

Reply:
Yes, we could. I was just following the pattern we have in Lit-LLaMA of having separate arguments.

Since this repo only uses data from HF, we could have `convert_hf_checkpoint.py` just create a `lit-checkpoint.pth` in the input weights dir, without having 2 directories.
Then you could just specify the link to the directory and you are good to go. What do you think?

This is related to #9 

Still, I propose that we do this in a separate PR as it will impact multiple files